$MirOS: ports/www/firesomething/patches/patch-security_coreconf_rules_mk,v 1.5 2008/12/05 16:26:11 bsiegert Exp $
--- security/coreconf/rules.mk.orig	Tue Aug 11 03:23:15 2009
+++ security/coreconf/rules.mk	Tue Mar 16 18:52:41 2010
@@ -338,8 +338,7 @@ ifdef MT
 endif	# MSVC with manifest tool
 endif
 else
-	$(MKSHLIB) -o $@ $(OBJS) $(SUB_SHLOBJS) $(LD_LIBS) $(EXTRA_LIBS) $(EXTRA_SHARED_LIBS) $(OS_LIBS)
-	chmod +x $@
+	$(MKSHLIB) -o $@ $(OBJS) $(SUB_SHLOBJS) $(LD_LIBS) $(EXTRA_LIBS) $(EXTRA_SHARED_LIBS) $(OS_LIBS) $(LDFLAGS)
 ifeq ($(OS_TARGET),Darwin)
 ifdef MAPFILE
 	nmedit -s $(MAPFILE) $@
